dev/Breadcrumb/Breadcrumb.h

```diff
@@ -0,0 +1,79 @@
+    public BreadcrumbProperties
+{
+
+public:
+    Breadcrumb();
+    ~Breadcrumb() {}
+
+    // IFrameworkElement
+    void OnApplyTemplate();
+    void OnPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args);
+
+    void RaiseItemClickedEvent(const winrt::IInspectable& content);
+    winrt::Collections::IVector<winrt::IInspectable> HiddenElements();
+
+private:
+    void OnBreadcrumbItemRepeaterLoaded(const winrt::IInspectable&, const winrt::RoutedEventArgs&);
+    void OnElementPreparedEvent(const winrt::ItemsRepeater&, const winrt::ItemsRepeaterElementPreparedEventArgs&);
+    void OnElementIndexChangedEvent(const winrt::ItemsRepeater&, const winrt::ItemsRepeaterElementIndexChangedEventArgs&);
+    void OnElementClearingEvent(const winrt::ItemsRepeater&, const winrt::ItemsRepeaterElementClearingEventArgs&);
+    void OnRepeaterCollectionChanged(const winrt::IInspectable&, const winrt::IInspectable&);
+
+    // Keyboard navigation
+    void OnGettingFocus(const winrt::IInspectable&, const winrt::GettingFocusEventArgs& args);
+    void OnAccessKeyInvoked(const winrt::UIElement&, const winrt::AccessKeyInvokedEventArgs& args);
+
+    winrt::FindNextElementOptions GetFindNextElementOptions();
+    void FocusElement(int index);
+    bool MoveFocus(int initialIndexIncrement);
+    bool MoveFocusPrevious();
+    bool MoveFocusNext();
+    bool HandleEdgeCaseFocus(bool first, const winrt::IInspectable& source);
+    void OnChildPreviewKeyDown(const winrt::IInspectable& sender, const winrt::KeyRoutedEventArgs& args);
+
+    void UpdateItemsSource();
+    void UpdateItemTemplate();
+    void UpdateDropdownItemTemplate();
+
+    winrt::IInspectable GenerateInternalItemsSource();
+    void ForceUpdateLastElement();
+    void UpdateLastElement(const winrt::BreadcrumbItem& newLastBreadcrumbItem);
+    
+    winrt::Control::Loaded_revoker m_breadcrumbItemRepeaterLoadedRevoker{};
+    winrt::ItemsRepeater::ElementPrepared_revoker m_itemRepeaterElementPreparedRevoker{};
+    winrt::ItemsRepeater::ElementIndexChanged_revoker m_itemRepeaterElementIndexChangedRevoker{};
+    winrt::ItemsRepeater::ElementClearing_revoker m_itemRepeaterElementClearingRevoker{};
+    winrt::ItemsSourceView::CollectionChanged_revoker m_itemsSourceChanged{};
+    winrt::IObservableVector<winrt::IInspectable>::VectorChanged_revoker m_itemsSourceChanged2{};
+
+    tracker_ref<winrt::INotifyCollectionChanged> m_notifyCollectionChanged{ this };
+    winrt::event_token m_collectionChanged{ };
+    winrt::ItemsSourceView m_breadcrumbItemsRepeaterItemsSource{ nullptr };
+
+    tracker_ref<winrt::ItemsRepeater> m_breadcrumbItemRepeater { this };
+    com_ptr<BreadcrumbElementFactory> m_breadcrumbElementFactory{ nullptr };
+
+    tracker_ref<winrt::BreadcrumbItem> m_ellipsisBreadcrumbItem { this };
+    tracker_ref<winrt::BreadcrumbItem> m_lastBreadcrumbItem { this };
+
+    // Index of the last focused item when breadcrumb lost focus
+    int m_focusedIndex{ 1 };
```

Comment:
I haven't found any issue, the element to be focused is retrieved with a TryGet method call but if it can't access the element then it doesn't try to do anything with the element